src/pages/Profile.js

```diff
@@ -173,79 +175,34 @@ const FenceLink = ({ provider, displayName }) => {
    * Hooks
    */
   const [{ username, issued_at: issuedAt }, setStatus] = useState({})
-  const [href, setHref] = useState(undefined)
-  const [isLoadingStatus, setIsLoadingStatus] = useState(false)
-  const [isLoadingAuthUrl, setIsLoadingAuthUrl] = useState(false)
   const [isLinking, setIsLinking] = useState(false)
   const signal = Utils.useCancellation()
 
   const { User } = Ajax(signal)
 
-  const loadAuthUrl = _.flow(
-    withErrorReporting(`Error loading Fence link`),
-    Utils.withBusyState(setIsLoadingAuthUrl)
-  )(async () => {
-    const result = await User.getFenceAuthUrl(provider, redirectUrl)
-    setHref(result.url)
-  })
-
-  const loadFenceStatus = _.flow(
-    withErrorReporting(`Error loading status for ${displayName}`),
-    Utils.withBusyState(setIsLoadingStatus)
-  )(async () => {
-    try {
-      setStatus(await User.getFenceStatus(provider))
-    } catch (error) {
-      if (error.status === 404) {
-        setStatus({})
-      } else {
-        throw error
-      }
-    }
-  })
-
-  const linkFenceAccount = _.flow(
-    withErrorReporting('Error linking NIH account'),
-    Utils.withBusyState(setIsLinking)
-  )(async () => {
+  const linkFenceAccount = Utils.withBusyState(setIsLinking)(async () => {
```

Comment:
Since the underlying link call can fail, I think we still want to keep the error reporting wrapper here.